### ISSUE TITLE
waiting for selector failed: timeout 5000ms exceeded

### ISSUE THREAD

**Issue by netzwerkwelten** [social-manager-tools/socialmanagertools-gui#65]
### Versions

Replace the values below with your own:

- **Social Manager Tools Version:** v0.6.4 // v0.9.14
- **Social Platform:** Instagram
- **Operating System:** Mac OS

### Expected Behavior

Please describe the program's expected behavior.

```
fixed error is back: 
TimeoutError: waiting for selector "article:nth-child(1) section:nth-child(1) button:nth-child(1)" failed: timeout 5000ms exceeded
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```

```

**Comment by ptkdev** [comment]
What mode you use? `Likemode_realistuic`? `Commentmode`?

**Comment by netzwerkwelten** [comment]
Hi,

I use Like Mode realitic and tested the classic mode too. Same error.



Patryk Rzucidło <notifications@github.com> schrieb am Di., 14. Mai 2019,
12:00:

**Comment by ptkdev** [comment]
@netzwerkwelten imho this is mac-only bug, i try on **linux** and **windows** and work. Stay tuned...

**Comment by ptkdev** [comment]
What chrome version you use? **v74**?

**Comment by netzwerkwelten** [comment]
I use the macos electron app in the latest version

Patryk Rzucidło <notifications@github.com> schrieb am Di., 14. Mai 2019,
15:01:

**Comment by ptkdev** [comment]
Yes, but what is version of `google chrome` installed in your macos? This app use chrome's api installed in your mac :)

**Comment by netzwerkwelten** [comment]
74.0.3729.131
64 bit

Patryk Rzucidło <notifications@github.com> schrieb am Di., 14. Mai 2019,
15:48:

**Comment by ptkdev** [comment]
Imho, fast solution is downgrade to **v73**. Thanks for you feedback and details.

I try fix this issue with chrome **v74** in next release.

**Comment by ptkdev** [comment]
Fixed in **v0.6.5**
